source/adam.cpp

```diff
@@ -1114,36 +1114,34 @@ void sheet_t::implementation_t::flow(cell_bits_t& priority_accessed) {
                 index_t::iterator iter = output_index_m.find(term->name_set_m[n]);
                 assert(iter != output_index_m.end());
 
-                cell_t& cell = *iter;
+                cell_t& e = *iter;
```

Comment:
OMG this is a hecka long function.  Nice getting rid of the type information in the name, but can we do better than `e`?  What role is this variable playing on the inside of the (raw!) loop?  That is, if you were to break the body of the loop into a function call taking `cell_t&`, what would you call the parameter?

Reply:
I did a fair amount of refactoring and cleanup of that function and used better names. Still long - opened Issue#127 to revisit.